### ISSUE TITLE
Maximum call stack size exceeded

### ISSUE THREAD

**Issue by RobinvandenHurk** [amcharts/amcharts4#1133]
Hi. I stumbled upon what appears to be a bug. Whenever i try to create a pie chart , and load it with values higher than 1000000 (million), I get a Maximum call stack size exceeded error. When the value is 999999, the chart loads without any trouble.

I create the pie chart like so:
    `creditorChart = am4core.create("topCred", am4charts.PieChart);`
    `updateCreditorData();`
    `var pieSeries = creditorChart.series.push(new am4charts.PieSeries());`
    `pieSeries.dataFields.value = "amount";`
    `pieSeries.dataFields.category = "name";`
    `pieSeries.ticks.template.disabled = true;`
    `pieSeries.alignLabels = false;`
    `pieSeries.labels.template.disabled = true;`
    `creditorChart.legend = new am4charts.Legend();`
    `creditorChart.legend.position = "left";`
    `creditorChart.legend.fontSize = 12;`
    `creditorChart.numberFormatter.numberFormat = "€#,###";`

The function `updateCreditorData()` simply makes a ajax request to retrieve the data, and loads it into the chart using `creditorChart.data = data`

I use the scripts of the CDN as follows:
`<script src="//www.amcharts.com/lib/4/core.js"></script>`
`<script src="//www.amcharts.com/lib/4/charts.js"></script>`

I tried removing the number formatting, but that didn't help.

The following appears in the console:

`RangeError: Maximum call stack size exceeded`
`at RegExp.[Symbol.replace] (<anonymous>)`
`at String.replace (<anonymous>)`
 `at a (core.js:1)`
 `at a (core.js:1)`
 `at a (core.js:1)`
 `at a (core.js:1)`
 `at a (core.js:1)`
 `at a (core.js:1)`
 `at a (core.js:1)`
 `at a (core.js:1)`

I have looked around, and couldn't find this same issue anywhere. I'm sorry if I overlooked something.

**Comment by RobinvandenHurk** [comment]
I just found out something interesting.

I noticed I'm putting in values with . as 1000 markers. When i push this data `[{name: "credi", amount: "25.201"}]` The chart gets displayed like normal. However, when i go over the million, the data looks like this: `[{name: "credi", amount: "1.000.201"}]` and I get that error. Seems like AmCharts doesn't like it when you supply more than one . in the value. After removing the thousants indicator in the values, the new data looks like this: `[{name: "credi", amount: 1000201}]`, and the chart displays the correct value like normal. I don't know if this is still concidered a bug then.
